redbadger/website-honestly
# Bump polyfill version

### Motivation

- Fixes bug #606 

On Chrome the Camden case study doesn't show the header animation.

### Test plan

- Go to Camden Market case study in Chrome https://www-staging.red-badger.com/2980ea2/our-work/case-study/camden-market/
- Observe that on desktop the header has the animation

### Pre-merge checklist
